.github/workflows/autoblack.yml

```diff
@@ -0,0 +1,35 @@
+name: Check / auto apply Black
+on:
+  push:
+      branches:
+          - master
+jobs:
+  black:
+    name: Check / auto apply black
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check files using the black formatter
+        uses: psf/black@stable
+        id: black
+        with:
+          options: "."
+        continue-on-error: true
+      - shell: pwsh
+        id: check_files_changed
+        run: |
+          # Diff HEAD with the previous commit
+          $diff = git diff
+          $HasDiff = $diff.Length -gt 0
+          Write-Host "::set-output name=files_changed::$HasDiff"
```

Comment:
wut what

Reply:
That's needed for the if in the next step to only do a merge request if files have been modified